<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rogner</class>
 <widget class="QDialog" name="rogner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>760</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>759</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>20</width>
     <height>450</height>
    </rect>
   </property>
   <property name="maximum">
    <number>449</number>
   </property>
   <property name="value">
    <number>449</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>70</y>
     <width>20</width>
     <height>450</height>
    </rect>
   </property>
   <property name="maximum">
    <number>449</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>760</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>759</number>
   </property>
   <property name="value">
    <number>759</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>ROGNER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>ENREGISTRER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>fermer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
